extensions/amp-video-iframe/0.1/amp-video-iframe.js

```diff
@@ -357,12 +357,27 @@ class AmpVideoIframe extends AMP.BaseElement {
   }
 
   /**
+   * Creates an IntersectionObserver to fire a single intersection event.
+   *
    * @param {number} messageId
    * @private
    */
   postIntersection_(messageId) {
-    const {time, intersectionRatio} = this.element.getIntersectionChangeEntry();
+    const intersectionObserver = new IntersectionObserver((entries) => {
```

Comment:
How often is this invoked? Maybe it'd be better to just set a single instance of InOb on this instance of an element and then the callback will be more or less immediate? If this is more preferred, is this call throttled?

Reply:
I followed the protocol on the newer thread for the new component: https://github.com/ampproject/amphtml/pull/31055#discussion_r520958803.

Its publisher-invoked and therefore I dont know how often it is called. If we end up merging a `getIntersection` or `measureIntersection => Promise` utility, we can use that